api/controllers/course.js

```diff
@@ -18,8 +18,25 @@ const getCoursesForCarousel = async (req, res) => {
   res.status(200).send(response);
 };
 
+const postSignatureUserCourse = async (req, res) => {
```

Comment:
Honestly, i can not understand name of this function, maybe it's better to write smth like `signUserToCourse` / `addCourseToUser`

Reply:
Done